ts-e2e/package.json

```diff
@@ -14,22 +14,22 @@
   "dependencies": {
     "@azure/event-hubs": "^1.0.8",
     "@types/chai": "^3.5.1",
-    "@types/debug": "4.1.2",
+    "@types/debug": "^4.1.1",
     "@types/mocha": "^2.2.41",
-    "@types/node": "^7.0.8",
-    "@types/uuid": "^2.0.29",
+    "@types/node": "^9.3.0",
+    "@types/uuid": "^3.4.3",
     "azure-iot-common": "^1.9.0-preview.0",
     "azure-iot-device": "^1.9.0-preview.0",
     "azure-iot-device-amqp": "^1.9.0-preview.0",
     "azure-iot-device-http": "^1.9.0-preview.0",
     "azure-iot-device-mqtt": "^1.9.0-preview.0",
     "azure-iothub": "^1.9.0-preview.0",
```

Comment:
Really out of sync.

Reply:
this is pierre's project and he's admitted that it's out of sync.  I'm not sure if I should be touching it at all.

---
In reply to: [270528223](https://github.com/Azure/azure-iot-sdk-node/pull/521#discussion_r270528223) [](ancestors = 270528223)